<launch>
	<node name="image_processing_node" pkg="eve_main" type="image_processing_node" output="screen">
	</node>

	<node name="motor_node" pkg="eve_main" type="motor_node" output="screen">
	</node>

	<node name="eve_main_node" pkg="eve_main" type="eve_main_node" output="screen">
	</node>
</launch>
